psi4/src/psi4/dfocc/dfocc.cc

```diff
@@ -33,6 +33,8 @@
 #include "defines.h"
 #include "psi4/libpsi4util/PsiOutStream.h"
 #include "psi4/liboptions/liboptions.h"
+#include "psi4/libpsi4util/process.h"
```

Comment:
Should this header go in `dfocc/manager.cc` so as to minimize its impact on the build?